sway-lib-std/src/address.sw

```diff
@@ -21,10 +22,46 @@ impl core::ops::Eq for Address {
 
 /// Functions for casting between the `b256` and `Address` types.
 impl From<b256> for Address {
-    fn from(bits: b256) -> Address {
-        Address { value: bits }
+    /// Casts raw `b256` data to an `Address`.
+    /// 
+    /// # Arguments
+    ///
+    /// * `bits`: [b256] - The raw `b256` data to be casted.
+    /// 
+    /// # Returns
+    ///
+    /// * [Address] - The newly created `Address` from the raw `b256`.
+    ///
+    /// # Examples
+    /// 
+    /// ```sway
+    /// use std::constants::ZERO_B256;
+    ///
+    /// fn foo() {
+    ///    let address = Address::from(ZERO_B256);
+    /// }
+    /// ```
+    fn from(bits: b256) -> Self {
```

Comment:
We should not change the actual code (even if its simple) in a pr for documentation updates